### ISSUE TITLE
using the front camera leads to inaccurate visualizations of key points

### ISSUE THREAD

**Issue by lgwds** [shaqian/flutter_realtime_detection#28]
when I use the front camera like this:

```
controller = new CameraController(
        widget.cameras[1],
        ResolutionPreset.high,
      );
```
... the pose estimation key points seem way off. Could you please let me know which parameter settings I need to adjust to make it work for the front camera as well?

**Comment by lgwds** [comment]
my PR fixes this issue: #30

**Comment by yogi00195** [comment]
> my PR fixes this issue: #30

Please mention what you did to debug it
